docs/guides/js-waku/use-waku-react.md

```diff
@@ -0,0 +1,356 @@
+---
+title: "Build React DApps Using @waku/react"
+---
+
+The [@waku/react](https://www.npmjs.com/package/@waku/react) package provides components and UI adapters to integrate `js-waku` into React applications effortlessly. This guide provides detailed steps for using `@waku/react` in your project.
```

Comment:
I think we probably can start mentioning  `@waku/sdk` instead of `js-waku` 

cc @fryorcraken